tools/actions_local_runner.py

```diff
@@ -12,9 +12,10 @@
 import fnmatch
 import shlex
 import configparser
-
 from typing import List, Dict, Any, Optional, Tuple, Union
 
+from tools import mypy_wrapper
```

Comment:
Should we use this?

```suggestion
from . import mypy_wrapper
```

Reply:
I tried that and it didn't work: https://github.com/pytorch/pytorch/runs/2558431639 any other suggestions?